features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1149,6 +1156,14 @@
                    timeout="{{cache.idp_cache_by_name.timeout}}"
                    capacity="{{cache.idp_cache_by_name.capacity}}"
                    isDistributed="false"/>
+            {% for cache in cache.manager %}
+             <Cache id="{{cache.id}}"
+                    name="{{cache.name}}"
+                    enable="{{cache.enable}}"
+                    timeout="{{cache.timeout}}"
+                    capacity="{{cache.capacity}}"
+                    isDistributed="{{cache.is_distributed}}"/>
```

Comment:
This is always false. Do we need to template this?

Reply:
Set to false and removed the template